src/main/kotlin/fi/metatavu/oss/api/impl/surveys/SurveyController.kt

```diff
@@ -64,7 +65,6 @@ class SurveyController {
      * Deletes a survey
      *
      * @param surveyEntity survey to delete
-     * @return uni with void
      */
     suspend fun deleteSurvey(surveyEntity: SurveyEntity) {
         surveyRepository.deleteSuspending(surveyEntity)
```

Comment:
Shouldn't DeviceSurvey objects that depend on this survey be removed too?

Reply:
It was thought that preventing users from deleting Surveys that are assigned to a device would be a good idea.

Resolving this by that conversation.